src/games/strategy/triplea/ai/proAI/util/ProSortMoveOptionsUtils.java

```diff
@@ -54,7 +54,8 @@ public int compare(final Map.Entry<Unit, Set<Territory>> o1, final Map.Entry<Uni
   }
 
   public static Map<Unit, Set<Territory>> sortUnitNeededOptions(final PlayerID player,
-      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap) {
+      final Map<Unit, Set<Territory>> unitAttackOptions, final Map<Territory, ProTerritory> attackMap,
```

Comment:
Good point, since that would be an API change. You could help the transition by assigning a multimap to be equal to the parameter, and then use the multi map in the body of the method.